### DISCUSSION TITLE
THROTTLE1_SET

### DISCUSSION THREAD

**Discussion by peerluk** [nguyenquyhy/Flight-Tracker-StreamDeck#151] [Ideas]
Hi,

I'm trying to create a A320 profile. Basically create a toga/flex/CL button. But I don't see a Throttle set event.
Is it possible to add it? I see it in Simconnect Or maybe I'm missing something.

Great work btw. I use this all the time now!

**Comment by peerluk** [comment]
No awnser at all?   :(

**Reply by nguyenquyhy** [reply]
Is that the name of the event? `THROTTLE1_SET`? and probably `THROTTLE2_SET` too?

**Reply by peerluk** [reply]
yes and probably the generic one too to set throttle for all of them at the same time : THROTTLE_SET I'm guessing

**Reply by jplewis01** [reply]
Did the Throttle_Set Commands ever get added? I just got my Honeycomb Bravo after 6 Months waiting and find I'm going to need get Flight Tracker creative to support the TBM-930 Throttle Quadrant after update 5 or 6 seems to have broken it the ability to cleanly and easily shift its modes. I have some specific parameter values to set the Throttle and Propeller params too.
